packages/taquito-tezbridge-wallet/src/taquito-tezbridge-wallet.ts

```diff
@@ -28,6 +28,13 @@ export class TezBridgeWallet implements WalletProvider {
     });
   }
 
+  setHost(host: string) {
```

Comment:
Can you add a TypeDoc comment for this method please?

Reply:
Yup, no worries. Added.